packages/data-encoding/data-encoding.0.1.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Nomadic Labs" ]
+homepage: "https://gitlab.com/nomadic-labs/data-encoding"
+bug-reports: "https://gitlab.com/nomadic-labs/data-encoding/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/data-encoding.git"
+license: "MIT"
+depends: [
+  "ocaml" { >= "4.07" }
+  "dune" { >= "1.7" }
```

Comment:
```suggestion
  "dune" { >= "1.11" }
```
According to the `dune-project` file this packages requires at least dune 1.11

Reply:
I went the other way around and downgraded `dune-lang` in `dune-project`.

Reply:
Sure, that also works!